ai-platform-unified/notebooks/official/pipelines/pipelines_intro_kfp.ipynb

```diff
@@ -0,0 +1,1109 @@
+{
```

Comment:
<p>"evaluation"</p><p>Did you mean compilation?</p><p> </p><p>For example.</p><p>The compilation of this component creates a task factory function</p><p> </p><p>Explain what a task factory function is.</p><p> </p><p>"you can" -&gt; "you" (TW guidelines, don't use can, could, may).</p><p> </p><p>"While we don't show it here, if " -&gt; If (TW guidelines, don't use we, us, let's)</p><p> </p><p>"(For GitHub URLs, load the 'raw' file.)" -&gt; *(For GitHub URLs, load the 'raw' file.* (TW guidelines, avoid parenthesis).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/335/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>